src/views/Transcript/index.js

```diff
@@ -19,6 +22,8 @@ export default class Transcript extends Component {
       loading: true,
       profile: {},
     };
+    //Variable used by SessionComparitor to determine when activies in the list are part of a new session
+    //Necessary in order to remember session info about previous item in list while looping though
```

Comment:
Thanks.  Would it make sense to name the variable "previousSessionVal" (or "prevSessionVal")?